### DISCUSSION TITLE
How to check the CRON?

### DISCUSSION THREAD

**Discussion by imknight** [directus/directus#17542] [Archive]
the cronjob has been running perfect daily, but it suddenly stop today? how do we check this node cron log ?

**Comment by liefra** [comment]
In the right sidebar of your flow configuration page there is a "Logs" panel which shows all cron runs, including errors. Do you see anything there?

**Reply by imknight** [reply]
nope, nothing show there, the log just stop at yesterday, so wonder whats wrong, is there any CLI command that I can access through terminal to check ?
